data/pigui/libs/wrappers.lua

```diff
@@ -0,0 +1,758 @@
+	for k,v in pairs(styles) do
+		pigui.PushStyleColor(k, v)
+	end
+	for k,v in pairs(vars) do
+		pigui.PushStyleVar(k, v)
+	end
+	local res = fun()
+	pigui.PopStyleVar(utils.count(vars))
+	pigui.PopStyleColor(utils.count(styles))
+	return res
+end
+
+--
+-- Function: ui.screenSize
+--
+-- ui.screenSize()
+--
+-- Return the current screen resolution as a Vector2
+--
+-- Example:
+--
+-- > 
+--
+-- Parameters:
+--
+-- Returns:
+--
+--   Vector2 - screen width and height
+--
+function ui.screenSize()
+	return Vector2(ui.screenWidth, ui.screenHeight)
+end
+
+--
+-- Function: ui.setNextWindowPosCenter
+--
+-- ui.setNextWindowPosCenter(cond)
+--
+-- Set the next window position to be centered on screen
+--
+-- Example:
+--
+-- > 
+--
+-- Parameters:
+--   cond - table, condition flags: Always, Once, FirstUseEver, Appearing
+--
+-- Returns:
+--
+--   nil
+--
+function ui.setNextWindowPosCenter(cond)
+	ui.setNextWindowPos(ui.screenSize() / 2, cond, Vector2(0.5, 0.5))
+end
+
+--
+-- Function: ui.sameLine
+--
+-- ui.sameLine(pos_x, spacing_w)
+--
+-- Draw the next command on the same line as the previous
+--
+-- Example:
+--
+-- > 
+--
+-- Parameters:
+--   pos_x     - (Optional) number, X position for next draw command, default 0
+--   spacing_w - (Optional) number, draw with spacing relative to previous, default -1
+--
+-- Returns:
+--
+--   nil
+--
+function ui.sameLine(pos_x, spacing_w)
+	local px = pos_x or 0.0
+	local sw = spacing_w or -1.0
+	pigui.SameLine(px, sw)
+end
+
+--
+-- Function: ui.withID
+--
+-- ui.withID(id, fun)
+--
+-- Display content with a specified ID
+--
+-- Example:
+--
+-- > 
+--
+-- Parameters:
+--   id  - string, the desired ID
+--   fun - function, function called to display content
+--
+-- Returns:
+--
+--   nil
+--
+function ui.withID(id, fun)
+		pigui.PushID(id)
+		fun()
+		pigui.PopID()
+end
+
+--
+-- Function: ui.loadTextureFromSVG
+--
+-- ui.loadTextureFromSVG(filename, width, height)
+--
+-- Create a texture from an SVG
+--
+-- Example:
+--
+-- > 
+--
+-- Parameters:
+--   filename - string, svg path
+--   width    - number, width of texture to create
+--   height   - number, height of texture to create
+--
+-- Returns:
+--
+--   userdata - the texture
+--
+function ui.loadTextureFromSVG(filename, width, height)
+	return pigui:LoadTextureFromSVG(filename, width, height)
+end
+
+--
+-- Function: ui.loadTexture
+--
+-- ui.loadTexture(filename)
+--
+-- Load a texture from file
+--
+-- Example:
+--
+-- > 
+--
+-- Parameters:
+--   filename - string, texture file path
+--
+-- Returns:
+--
+--   userdata - the texture
+--
+function ui.loadTexture(filename)
+	return pigui:LoadTexture(filename)
+end
+
+
+-- this doesn't appear to be used here...
+--local function maybeSetTooltip(tooltip)
+--	if not Game.player:IsMouseActive() then
+--		pigui.SetTooltip(tooltip)
+--	end
+--end
+
+function ui.registerTheme(name, theme)
+	assert(type(theme) == "table", "UI Themes must be table values!")
+	pigui.handlers[name] = theme
+end
```

Comment:
This should go in the same file with `ui.registerHandler()` - they serve the same purpose.

Reply:
Ok, both this and `ui.registerHandler` are now in baseui.lua.